simulator_launch/launch/simulator.launch.xml

```diff
@@ -24,6 +25,46 @@
     </include>
   </group>
 
+  <!-- perception module -->
+  <group>
+    <push-ros-namespace namespace="perception"/>
+    <!-- object recognition -->
+    <group if="$(var perception/enable_object_recognition)">
+      <push-ros-namespace namespace="object_recognition"/>
+      <!-- tracking module -->
+      <group>
+        <push-ros-namespace namespace="tracking"/>
+        <include file="$(find-pkg-share perception_launch)/launch/object_recognition/tracking/tracking.launch.xml">
+        </include>
+      </group>
+      <!-- prediction module -->
+      <group>
+        <push-ros-namespace namespace="prediction"/>
+        <include file="$(find-pkg-share perception_launch)/launch/object_recognition/prediction/prediction.launch.xml">
+          <arg name="use_vector_map" value="true" />
+        </include>
+      </group>
+    </group>
+
+    <!-- object segmentation module -->
+    <group>
+      <push-ros-namespace namespace="occupancy_grid_map"/>
+      <!-- Occupancy Grid -->
+      <include file="$(find-pkg-share laserscan_to_occupancy_grid_map)/launch/laserscan_to_occupancy_grid_map.launch.py">
```

Comment:
@yukkysaito 
Should this be as following?
`      <include file="$(find-pkg-share perception_launch)/launch/occupancy_grid_map/occupancy_grid_map.launch.py">`

Reply:
I just copied from here, so you can change as you want.
https://github.com/tier4/autoware.universe/blob/920b874ad74283d0ba828b9e5a80250e44d130a6/simulator/dummy_perception_publisher/launch/dummy_perception_publisher.launch.xml#L56-L64

Reply:
I couldn't sync this PR with the following PR, since the following one was merged one day before this PR, and I didn't notice the following PR at that time. :pray: 
https://github.com/tier4/autoware.universe/commit/fcfcceda3f0227b3393ced3935981f592d890b4e